samples/net/https_client/README.rst

```diff
@@ -0,0 +1,124 @@
+.. _https-client-sample:
+
+HTTPS Client
+############
+
+Overview
+********
+
+This sample application shows how to create encrypted HTTP 1.1 requests
+to an HTTPS server and how to parse the incoming responses.
+Supported HTTP 1.1 methods are: GET, HEAD, OPTIONS and POST.
+
+The source code for this sample application can be found at:
+:file:`samples/net/https_client`.
+
+Requirements
+************
+
+- :ref:`Networking with Qemu <networking_with_qemu>`
+- Terminal emulator software
+- HTTPS Server
+- DNS server (optional)
+
+Building and Running
+********************
+
+Open the project configuration file for your platform, for example:
+:file:`prj_qemu_x86.conf` is the configuration file for QEMU.
+
+To use QEMU for testing, follow the :ref:`Networking with Qemu
```

Comment:
Ouch, I copied the ref from some old sample which obviously was wrong already.
I will fix all invocations of those incorrectly typed refs in PR #415